htdocs/web_portal/controllers/downtime/add_downtime.php

```diff
@@ -158,62 +158,103 @@ function draw(\User $user = null) {
         die(json_encode(array('UTC', 0)));
     }
 
-    // URL Mapping
-    // If the user wants to add a downtime to a specific site, show only that site's SEs
-    else if(isset($_GET['site'])) {
-        $site = \Factory::getSiteService()->getSite($_GET['site']);
-        if(\Factory::getRoleActionAuthorisationService()->authoriseAction(\Action::EDIT_OBJECT, $site, $user)->getGrantAction() == FALSE){
-           throw new \Exception("You don't have permission over $site");
-        }
-        $ses = $site->getServices();
-        $params = array('ses' => $ses, 'nowUtc' => $nowUtcDateTime->format('H:i T'), 'selectAll' => true);
-        show_view("downtime/add_downtime.php", $params);
-        die();
+    // URL Mapping for `site` and `se`
+    elseif (isset($_GET['site']) && isset($_GET['se'])) {
+        displaySiteAndSeEndpoints($user, $nowUtcDateTime);
     }
 
-    // URL Mapping
-    // If the user wants to add a downtime to a specific SE, show only that SE
-    else if(isset($_GET['se'])) {
-        $se = \Factory::getServiceService()->getService($_GET['se']);
-        $site = \Factory::getSiteService()->getSite($se->getParentSite()->getId());
-        if(\Factory::getRoleActionAuthorisationService()->authoriseAction(\Action::EDIT_OBJECT, $se->getParentSite(), $user)->getGrantAction() == FALSE){
-           throw new \Exception("You do not have permission over $se.");
-        }
+    // URL Mapping for `site` ONLY
+    elseif (isset($_GET['site'])) {
+       displaySiteEndpoints($user, $nowUtcDateTime);
```

Comment:
```suggestion
        displaySiteEndpoints($user, $nowUtcDateTime);
```
indent only 7 spaces, not 8